qa/rpc-tests/bip68-sequence.py

```diff
@@ -235,7 +237,8 @@ def test_nonzero_locks(orig_tx, node, relayfee, use_height_lock):
             tx.nVersion = 2
             tx.vin = [CTxIn(COutPoint(orig_tx.sha256, 0), nSequence=sequence_value)]
             tx.vout = [CTxOut(int(orig_tx.vout[0].nValue - relayfee*COIN), CScript([b'a']))]
-            tx.rehash()
+            bloat_tx(tx)
+            tx.calc_sha256()
```

Comment:
Why switch from tx.rehash() to tx.calc_sha256()?  They're effectively equal here (and the other places where this change was made)

Reply:
It's just a small optimization, `calc_sha256` only calculates the hash if it has not been done before. I know bloat_tx does a rehash, but I don't want to depend on this side effect.